File: 프로그래머스/0/181858. 무작위로 K개의 수 뽑기/무작위로 K개의 수 뽑기.java
import java.util.*;
class Solution {
    public int[] solution(int[] arr, int k) {
      ArrayList<Integer> result = new ArrayList<>();
      
        for(int i=0; i<arr.length; i++){
            if(result.size() <k){
                if(!result.contains(arr[i])){
                result.add(arr[i]);
                }
            }
        }
        
        while(result.size()<k){
            result.add(-1);
        }
        
        int [] answer = new int[result.size()];
        for(int i =0; i<result.size(); i++){
            answer[i] = result.get(i);
        }
        return answer;
    }
}

File: 프로그래머스/0/181858. 무작위로 K개의 수 뽑기/무작위로 K개의 수 뽑기.js
function solution(arr, k) {
    const result = [];
    
    for (let i = 0; i < arr.length; i++) {
        if (result.length < k) {
            if (!result.includes(arr[i])) {
                result.push(arr[i]);
            }
        }
    }
    
    while (result.length < k) {
        result.push(-1);
    }
    
    return result;
}
 